tests/test_languages.py

```diff
@@ -465,15 +465,15 @@ def setUp(self):
         param('kk', "жексенбі 12 қыркүйек 1890", "sunday 12 september 1890"),
 
         # kl
-        param('kl', "2 martsi 2001 ataasinngorneq", "2 march 2001 monday"),
-        param('kl', "pin 1 oktoberi 1901", "wednesday 1 october 1901"),
+        param('kl', "2 marsi 2001 ataasinngorneq", "2 march 2001 monday"),
+        param('kl', "pin 1 oktobari 1901", "wednesday 1 october 1901"),
 
         # kln
         param('kln', "3 ng'atyaato koang'wan 10:09 kooskoliny", "3 february thursday 10:09 pm"),
         param('kln', "kipsuunde nebo aeng' 14 2009 kos", "december 14 2009 wednesday"),
 
         # kok
-        param('kok', "1 नोव्हेंबर 2000 आदित्यवार 01:19 मनं", "1 november 2000 sunday 01:19 pm"),
+        param('kok', "1 नोव्हेंबर 2000 आयतार 01:19", "1 november 2000 sunday 01:19"),
```

Comment:
Konkani (Indian language):  Can't be verified but new CLDR 39 data updates this.